org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/wizard/project/ProjectPreviewWizardPageUiTest.groovy

```diff
@@ -0,0 +1,76 @@
+package org.eclipse.buildship.ui.wizard.project
+
+import org.eclipse.swtbot.eclipse.finder.waits.Conditions
+import org.eclipse.core.runtime.NullProgressMonitor
+import org.eclipse.buildship.core.CorePlugin
+import org.eclipse.buildship.core.Logger
+import org.eclipse.buildship.ui.SWTBotTestHelper
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotShell
+import org.eclipse.swtbot.eclipse.finder.SWTWorkbenchBot
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import spock.lang.Specification
+import org.eclipse.buildship.ui.test.fixtures.TestEnvironment
+import org.eclipse.core.runtime.jobs.Job
+
+
+class ProjectPreviewWizardPageUiTest extends Specification {
+    SWTWorkbenchBot bot = SWTBotTestHelper.getBot()
+
+    @Rule
+    TemporaryFolder tempFolder
+    File location
+
+    def setup() {
```

Comment:
don't you want to screenshot the thing if the test fail or was it punted to later?